aleph/settings.py

```diff
@@ -132,9 +132,6 @@ def __init__(self) -> None:
         # Document processing pipeline
         self.INGEST_PIPELINE = env.to_list("ALEPH_INGEST_PIPELINE", ["analyze"])
 
-        # Result high-lighting
-        self.RESULT_HIGHLIGHT = env.to_bool("ALEPH_RESULT_HIGHLIGHT", True)
-
```

Comment:
Is there a particular reason for removing this global configuration option? Some Aleph admins might prefer to disable highlighting altogether due to high resource usage.